CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * [FEATURE] Store-gateway: streaming of series. The store-gateway can now stream results back to the querier instead of buffering them. This is expected to greatly reduce peak memory consumption while keeping latency the same. You can enable this feature by setting `-blocks-storage.bucket-store.batch-series-size` to a value in the high thousands (5000-10000). This is still an experimental feature and is subject to a changing API and instability. #3540 #3546 #3587 #3606 #3611 #3620 #3645 #3355 #3697 #3666 #3687 #3728 #3739 #3751 #3779 #3839
 * [FEATURE] Alertmanager: Added support for the Webex receiver. #3758
 * [FEATURE] Limits: Added the `-validation.separate-metrics-group-label` flag. This allows further separation of the `cortex_discarded_samples_total` metric by an additional `group` label - which is configured by this flag to be the value of a specific label on an incoming timeseries. Active groups are tracked and inactive groups are cleaned up on a defined interval. The maximum number of groups tracked is controlled by the `-max-separate-metrics-groups-per-user` flag. #3439
+* [FEATURE] Overrides-exporter: Added experimental ring support to overrides-exporter via `overrides-exporter.ring.enabled`. When enabled, the ring is used to shard tenants to overrides-exporters and avoid export of duplicate per-tenant limit override metrics. #3908
```

Comment:
[nit] We're used to prefix CLI flags with `-` to make it more clear they're flags.

```suggestion
* [FEATURE] Overrides-exporter: Added experimental ring support to overrides-exporter via `-overrides-exporter.ring.enabled`. When enabled, the ring is used to shard tenants to overrides-exporters and avoid export of duplicate per-tenant limit override metrics. #3908
```